src/casp/services/api/data_manager/cell_quota.py

```diff
@@ -38,3 +38,20 @@ def get_cells_processed_this_week_for_user(self, user: models.User) -> int:
         with self.system_data_db_session_maker() as session:
             cell_count_sum_query = sa.sql.text(self.SQL_GET_CELLS_PROCESSED_THIS_WEEK)
             return session.execute(cell_count_sum_query, {"id": user.id, "start_of_week": start_of_week}).first()[0]
+
+    def increase_quota_for_user(self, user: models.User, new_quota: int):
+        """
+        Increase cell processing quota for a specific user
+
+        :param user: User object to increase quota for
+        :param new_quota: The new quota to set for the user
+
+        :return: User object with updated quota
```

Comment:
Seems like this method doesn't return anything. I guess we don't need description for return?

But I'd put `-> None` in the method return annotation like this:
`def increase_quota_for_user(self, user: models.User, new_quota: int) -> None:`

Reply:
Oh oops.  I originally had it returning the user object, but changed my mind.  I'll fix that